modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/GridDhtLockFuture.java

```diff
@@ -1154,6 +1157,11 @@ private void loadMissingFromStore() {
      * Lock request timeout object.
      */
     private class LockTimeoutObject extends GridTimeoutObjectAdapter {
+        /** */
+        final long longOpDumpTimeout = IgniteSystemProperties.getLong(
```

Comment:
Adding long field overhead to every timeout object holding a constant looks wrong.
Better add new property to IgniteTxManager and create getter/setter. For example look at org.apache.ignite.internal.processors.cache.transactions.IgniteTxManager#slowTxWarnTimeout.
DFLT_LONG_OPERATIONS_DUMP_TIMEOUT will be not needed in such case (could be defined in IgniteTxManager).